helm_deploy/sentence-planning/templates/service.yaml

```diff
@@ -8,12 +8,11 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
-  type: NodePort
+  type: ClusterIP
```

Comment:
I don't think we should be using NodePort here so changed to use the default ClusterIP